kannon/master.py

```diff
@@ -56,6 +55,26 @@ def __init__(
         self.task_id_to_job_name: dict[str, str] = dict()
 
     def build(self, root_task: gokart.TaskOnKart) -> None:
+        # TODO: support multiple dynamic config files
```

Comment:
If you like to support this, I think it is better to change the parameter name plural form ( `dynamic_config_paths` ) for compatibility. And then you assert like following

```python
assert len(dynamic_config_paths) == 1, 'Currently kannon doesn't support multiple dynamic config files'
```